configure.ac

```diff
@@ -533,6 +537,9 @@ if test "$ac_cv_lib_json_c_json_object_get" = no; then
 fi
 ])
 
+AC_ARG_ENABLE([command_log_disable],
+    AS_HELP_STRING([--enable_command_log_disable], [Dissallow the `no log commands` command]))
+
```

Comment:
To record the state here, I had suggested a command line option, and meant a daemon startup time option with that (e.g. `--command-log-always`).  Adding configure options is generally not a good idea and really not warranted for this one.